### PR TITLE
Add image utils

### PR DESCRIPTION
**Pull request by soltysh**
This is very small fraction of image helpers that is being shared between apiserver, controllers and oc.

/assign @deads2k

### CODE REVIEW

File: pkg/image/helpers.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package image
```
Comment (deads2k) [inline pkg/image/helpers.go]: move to a subpackage.  even pkg/image/imageutil is swallowable.

Comment (soltysh) [reply pkg/image/helpers.go]: Done

File: pkg/image/trigger/pods.go [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+	}
+	return nil, errNoImageMutator
+}
+
+var errNoPodSpec = fmt.Errorf("No PodSpec available for this object")
+
+// GetPodSpecV1 returns a mutable pod spec out of the provided object, including a field path
+// to the field in the object, or an error if the object does not contain a pod spec.
+// This only returns pod specs for v1 compatible objects.
+func GetPodSpecV1(obj runtime.Object) (*corev1.PodSpec, *field.Path, error) {
```
Comment (deads2k) [inline pkg/image/trigger/pods.go]: keep in package like imagereferencemutator.  We want these contained.

**Pull request merged**